src/network/networkd-manager.c

```diff
@@ -1444,6 +1445,19 @@ int manager_new(Manager **ret, sd_event *event) {
 
         (void) routing_policy_load_rules(m->state_file, &m->rules_saved);
 
+        /* Router lifetime can be set with netlink interface since kernel >= 4.5
+         * so for the supported kernel we dont need to expire routes in userspace */
+        {
+          Condition c = { .type = CONDITION_KERNEL_VERSION };
+          c.parameter = strdup(">= 4.5");
```

Comment:
why strdup this? why not just do: 

```c
Condition c = {
        .type = CONDITION_KERNEL_VERSION,
        .parameter = (char*) ">= 4.5",
};
```

Reply:
... which is what I wrote in my original comment ;)